discovery-provider/plugins/notifications/src/__tests__/processEmailNotifications.test.ts

```diff
@@ -431,6 +432,10 @@ describe('Email Notifications', () => {
     }
     await insertNotifications(discoveryDB, [notificationRow])
 
+    // Set last notification seen to two days ago
```

Comment:
nice this makes sense to me why it was needed. I think the only thing I would suggest is setting this to the beginning of time (a la `const epochDate = new Date(0);`) because I think we just need a value and the 2 days looks specific